modules/custom/az_core/az_core.module

```diff
@@ -52,6 +52,30 @@ function az_core_toolbar() {
     '#weight' => 101,
   ];
 
+  $siteurl = \Drupal::request()->getHost();
+
+  $items['az_dashboard'] = [
+    '#type' => 'toolbar_item',
+    'tab' => [
+      '#type' => 'link',
+      '#title' => t('Analytics Dashboard'),
+      '#url' => Url::fromUri('https://datastudio.google.com/u/0/reporting/938c3f2a-6dbf-4968-9edd-8963450b4200/page/kTLWB', [
+        'query' => [
+          'params' => '{"df62":"include%EE%80%800%EE%80%80IN%EE%80%80' . $siteurl . '/"}',
+        ],
+        'absolute' => TRUE,
+      ]),
+      '#attributes' => [
+        'target' => t('_blank'),
+        'title' => t('Analytics Dashboard'),
+        'class' => [
+          'toolbar-item',
+        ],
+      ],
+    ],
+    '#weight' => 102,
+  ];
```

Comment:
I _think_ this should work to add links as child items of the existing `Quickstart 2` toolbar item.

```suggestion
  $items['az_quickstart']['tray'] = [
    '#heading' => t('Quickstart links'),
    'az_links' => [
        '#theme' => 'links__toolbar_az_core',
        '#links' => [
          'docs' => [
            'title' => t('Documentation'),
            'url' => Url::fromUri('https://quickstart.arizona.edu'),
            'attributes' => [
              'title' => t('Arizona Quickstart 2 Documentation'),
              'target' => t('_blank'),
            ],
          ],
          'settings' => [
            'title' => t('Settings'),
            'url' => Url::fromRoute('az_core.az_quickstart'),
            'attributes' => [
              'title' => t('Arizona Quickstart 2 Settings'),
            ],
          ],
          'analytics' => [
            'title' => t('Analytics Dashboard'),
            'url' => Url::fromUri('https://datastudio.google.com/u/0/reporting/938c3f2a-6dbf-4968-9edd-8963450b4200/page/kTLWB', [
              'query' => [
                'params' => '{"df62":"include%EE%80%800%EE%80%80IN%EE%80%80' . $siteurl . '/"}',
              ],
              'absolute' => TRUE,
            ]),
            'attributes' => [
              'title' => t('Site Anayltics Dashboard'),
              'target' => t('_blank'),
            ],
          ],
        ],
        '#attributes' => [
          'class' => ['toolbar-menu'],
        ],
    ],
  ];
```